src/components/Navbar.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Navbar = () => {
+  return (
+    <nav className='nav'>
+        <Link to='/' className='nav-title'>Math Magicians</Link>
+        <ul>
+            <li><Link to='/home'>Home</Link></li>
+            <li><Link to='/calculator'>Calculator</Link></li>
+            <li><Link to='/quote'>Quote</Link></li>
+        </ul>
+    </nav>
+  )
+}
+
+export default Navbar
```

Comment:
- [ ] It would be better if we run the app without eslint errors as shown below

![Screenshot from 2022-08-16 22-39-28](https://user-images.githubusercontent.com/46388113/184998246-d83c210d-1390-4f0d-8330-3f327f1cde95.png)

Also in the terminal we could see some errors that needs to be fixed :+1:  as shown below

![Screenshot from 2022-08-16 22-42-58](https://user-images.githubusercontent.com/46388113/184998375-f0eabfa0-0211-48e5-8459-25793a4c9110.png)